ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml

```diff
@@ -293,15 +293,6 @@ presubmits:
         - --target=e2e-gcp-ovn-upgrade
         command:
         - ci-operator
-        env:
```

Comment:
@vrutkovs , I wonder if you could help me here. We noticed that our presubmit job for gcp-ovn-upgrade
was not using OVN. I can't figure out how 'make jobs' would create these, so maybe they are all
handcrafted and we no longer need to pass these env vars? that's the only major difference I can see
in this job config vs the periodic version of the same job which does deploy with OVN.

examples:

this is a periodic job which does deploy ovn:
https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-e2e-gcp-ovn-upgrade-4.7-stable-to-4.8-ci/1366818054662524928

here is the presubmit version which I thought would be similar and does not deploy ovn:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_ovn-kubernetes/447/pull-ci-openshift-ovn-kubernetes-master-e2e-gcp-ovn-upgrade/1365410443035676672

I'm not sure this is even the right thing to do, but it was all that stood out to me at this point.

Reply:
Run `make jobs`, it should reformat `ci-operator/jobs` automatically

Reply:
I tried. it does nothing, but where in config/ does it know that I want a pull-ci (presubmit) job? that's where I'm sort of
lost :(

Are presubmits auto-created with make jobs?

Reply:
>where in config/ does it know that I want a pull-ci (presubmit) job?

This config autocreates pull-ci presubmits and promote- postsubmits. Manual intervention in presubmits is not required, so if the script creates a presubmit with envs I suppose its valid (some might not be used anyway)

Reply:
I finally figured out what was wrong. The ovn-kubernetes config files were using a @daily cron for the job I was looking at which means it would end up as a periodic and not affecting the presubmits that I was trying to fix. I think the latest is now correct. I updated the commit message to explain what I'm doing.